cmd/helm/root.go

```diff
@@ -206,6 +216,9 @@ func newRootCmd(actionConfig *action.Configuration, out io.Writer, args []string
 	// Check for expired repositories
 	checkForExpiredRepos(settings.RepositoryConfig)
 
+	// CPU and memory profiling flags that are available to all commands
+	addProfilingFlags(cmd)
```

Comment:
(mostly duplicated from other comment)

Do we really want to have these flags, which are only for certain, advanced use-cases. Exposed to every user who runs `helm cmd --help`?

The way I think about it, is pprof is an advanced usage. And (newbie) users who run `helm cmd --help` will have the pprof flags presented, and this just adds to the things they need to parse and understand.

(so that said, flags I guess would be okay if they were hidden to the casual user)